app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def flash_class(level)
+    case level
+    when 'notice' then "alert alert-info"
+    when 'success' then "alert alert-success"
+    when 'alert' then "alert alert-warning"
+    when 'error' then "alert alert-danger"
+    end
+  end
```

Comment:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashLikeCase

Reply:
changed to hash